src/components/AccountSettings/__tests__/ChangePasswordForm.js

```diff
@@ -190,9 +190,9 @@ describe('ChangePasswordForm', () => {
 
     triggerInputChange(confirmNewPasswordInput, newPassword);
 
-    // Check if set button is present and disabled
```

Comment:
Interesting! so this test was actually never really working right?

Reply:
Exactly 😅